tests/rptest/tests/transactions_test.py

```diff
@@ -79,6 +79,39 @@ def consume_records():
                                  backoff_sec=2,
                                  err_msg="Can not consume data")
 
+    @cluster(num_nodes=3)
+    def find_coordinator_inits_tx_registry_test(self):
```

Comment:
kinda seems like the testing could be expanded quite a bit given we have now an entirely new service it seems. any plans to do that?

Reply:
Sure. There will be more tests ahead. Although this isn't new service, this is new stm for an existing service.

find coordinator handler already redirected a request to the tx registry which used static hashing partitioning similar to consumer groups. Now it persists static hashing on the first request and then uses persisted hashing for routing.

So tx registry is already covered with several tests:

- find_coordinator_creates_tx_topics_test
- init_transactions_creates_eos_topics_test
- find_coordinator_on_old_node_doesnt_create_tx_registry_test
- find_coordinator_on_new_node_doesnt_create_tx_registry_test
- find_coordinator_creates_tx_topics_after_upgrade_test

Besides, this PR's changes are on the hot path so other ducktape and chaos tests cover it too. Once the migration is supported we'll have more tests exercising it.